pkg/common/specification_types.go

```diff
@@ -24,8 +24,7 @@
 
 package common
 
-// IATMatrix - columns are minutes, rows are IATs
-type IATMatrix [][]float64
+type IATArray []float64
```

Comment:
I think we should create an explanation how exactly it is interpreted. For example, should we expect invocation before the first IAT, how would it correspond to the PerMinuteCounts. I think some inconsistencies in the tests and code come from that misunderstanding between how it was before (with shift option for example) and now.